src/app/styles/index.postcss.css

```diff
@@ -194,6 +194,10 @@ article code.language-plaintext {
   @apply bg-sc-gray-3 text-sc-gray-2 font-mono px-2 py-1 rounded;
 }
 
+body {
+  @apply break-all sm:break-normal;
```

Comment:
I would only apply this on the content of the page (not on the menu, header, footer, ...).

Reply:
At first I was not a fan (it is a not specific enough change / it can bleed a lot / we do not have visual tests to ensure no side effect). Like Jean said, I wish to have it set on a more specific target, for the rest I am considering it could the change needed, but I would like to be sure it has been extensively tested, especially by resizing the windows from mobile to 2xl screens

Reply:
Moved it to article and corrected the readme with Brandon and Pierre's informations !